README.md

```diff
@@ -25,7 +25,10 @@ The main privacy features on the network level:
 and on the blockchain level:
 - Intuitive ZeroLink CoinJoin integration.
 - Superb coin selection and labeling.
+- Opt-in PayJoin (BIP-78)
 - Dust attack protections.
+- Custom change address.
+- Anti Wallet/Transaction fingerprinting
```

Comment:
```suggestion
- Anti wallet fingerprinting.
```

fix capitalization, full stop at end, and I don't think it is called `transaction fingerprinting`...

Other than this nit, ACK.